config/initializers/friendly_id.rb

```diff
@@ -1,6 +1,12 @@
 FriendlyId.defaults do |config|
-  config.base = :name
-  config.use :slugged, :finders, FriendlyId::SequentialSlugs
+  module CustomNormalisation
+    def normalize_friendly_id(input)
+      # Use the babosa gem for better string parameterisation
+      super input.to_s.to_slug.truncate(150).normalize.to_s
+    end
+  end
 
+  config.base = :name
+  config.use :sequentially_slugged, :finders, CustomNormalisation
```

Comment:
If all the calls were changed, then theoretically, the `:finders` module is no longer required. That feels like a separate concern to the changes in this pull request though - the above changes don't add it, they merely maintain it.